team_admin/discord.md

```diff
@@ -0,0 +1,30 @@
+---
+layout: page
+title: Discord Server
+---
+
+# Discord Server
+
+To assist with communications, especially in a virtual competition, there's a [Discord](https://discord.com/) communication both for communication with blue-shirts and among your team.
```

Comment:
```suggestion
To assist with communications, especially in a virtual competition, there's a [Discord](https://discord.com/) chat server both for communication with blue-shirts and among your team.
```

Reply:
Can't suggest a change for some reason:

> To assist with communications, especially in a virtual competition, there's a [Discord](https://discord.com/) chat server both for communication with blue-shirts and among your team.

To assist with communications, especially in a virtual competition, there's a [Discord](https://discord.com/) chat server for communication both with blue-shirts and among your team.

Also, I'd get rid of another 1 of those "communication"'s